pkg/plugin/test-keeper/plugin/comment_message.go

```diff
@@ -10,17 +10,18 @@ import (
 )
 
 const (
-	beginning = "There wasn't found any test file that would be changed in this PR. Please add a test case to this PR.\n" +
+	beginning = "Thanks for this contribution! It seems that there are no tests that would be added or changed in this PR. " +
+		"Automated tests give us confidence in shipping reliable software. Please add some as part of this change.\n" +
 		"If you are an admin and you are sure that no test is needed then you can use command `" + SkipComment + "` " +
 		"as a comment to make the status green.\n"
 
 	noConfig = "For more information about the plugin and how to configure it, go to the " +
 		"[documentation](http://arquillian.org/ike-prow-plugins/#_test_keeper_plugin)."
 
-	withConfig = "To check plugin configuration that is set for your repository, please see the [configuration file](%s)."
+	withConfig = "To check your plugin configuration check [configuration file](%s)."
```

Comment:
there is the `check` word twice - it looks a bit weird to me

Reply:
Sure, somehow didn't notice. Thx